keymanager/src/client/remote.rs

```diff
@@ -218,6 +239,85 @@ impl RemoteClient {
         key.verify(self.runtime_id, key_pair_id, epoch, now, pk)
             .map_err(KeyManagerError::InvalidSignature)
     }
+
+    async fn churp_recover_state_key<S: Suite>(
+        &self,
+        key_id: KeyPairId,
+        status: churp::Status,
+    ) -> Result<StateKey, KeyManagerError> {
+        // Fault detection and blame assignment are not supported,
```

Comment:
Is this planned? If so, open an issue to track this.

Reply:
No. At least not in the near future.